Compiler/src/tfuncs.jl

```diff
@@ -2347,6 +2362,7 @@ const _EFFECT_FREE_BUILTINS = [
     isa,
     UnionAll,
     getfield,
+    Core.memorynew,
```

Comment:
Why is `memoryrefnew` available in Base by default, but `memorynew` is not?

Reply:
didn't seem needful to import.